backend/substrapp/migrations/0012_alter_algo_description_alter_algo_file_and_more.py

```diff
@@ -39,7 +39,7 @@ class Migration(migrations.Migration):
             field=models.FileField(
                 max_length=36,
                 storage=substrapp.storages.minio.MinioStorage("substra-compute-task-logs"),
-                upload_to=substrapp.models.compute_task_failure_report._upload_to,
+                upload_to=substrapp.models.asset_failure_report._upload_to,
```

Comment:
OK I'm really not familiar with migrations, is it expected that we modify _past_ migrations, or is this an unfortunate consequence of IDE renaming?

Reply:
I do think Django use the current model (the only one it has) in this kind of scenarios